tests/unit/cache/test_postgres_cache.py

```diff
@@ -166,62 +159,10 @@ def test_insert_or_append_operation_first_item(mock_connect):
     mock_cursor.execute.assert_has_calls(calls, any_order=True)
 
 
-@patch("psycopg2.connect")
```

Comment:
not needed anymore?

Reply:
I hope I didn't understood the test wrongly, but it seemed like it is a test for adding multiple entires via single `insert_or_append`, which is not possible anymore - only single entry can be added via `insert_or_append`.

Reply:
it checks if `insert_or_append` appends new Q&A into the thread. It must call `SELECT` followed by `INSERT`

Reply:
https://app.codecov.io/gh/openshift/lightspeed-service/pull/1159/blob/ols/src/cache/postgres_cache.py

Reply:
added back